pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -1053,10 +1053,12 @@ self: super: builtins.intersectAttrs super {
   hint = dontCheck super.hint;
 
   # Make sure that Cabal 3.8.* can be built as-is
-  Cabal_3_8_1_0 = doDistribute (super.Cabal_3_8_1_0.override {
+  Cabal_3_8_1_0 = doDistribute (super.Cabal_3_8_1_0.override ({
     Cabal-syntax = self.Cabal-syntax_3_8_1_0;
+  } // lib.optionalAttrs (lib.versionOlder self.ghc.version "9.2.5") {
+    # Use process core package when possible
     process = self.process_1_6_16_0;
-  });
+  }));
```

Comment:
Is there a good reason to have this override in configuration-nix?

Reply:
No particular reason, just was there before. I guess these kinds of overrides are a gray area.

Reply:
I agree. I’m fine either way.